website/docs/config.md

```diff
@@ -349,6 +350,47 @@ Here's an example of how you can customize the PR name template:
 pr_name = "release: {{ package }} {{ version }}"
 ```
 
+#### The `pr_body` field
+
+[Tera template](https://keats.github.io/tera/docs/#templates) of pull request's body that
+release-plz creates.
+
+By default it contains the summary of package updates, the changelog for each package, a section
+for breaking changes, and a footer with credits for release-plz. The text is trimmed to a length
+of 65536, because that's the limit imposed by Github.
+
+Here is an example of how you can customize the PR body template:
+
+```toml
+[workspace]
+pr_body = """
+{% for release in releases %}
+{% if release.title %}
+### {{release.title}}
+{% endif %}
+Package: {{release.package}} {{release.previous_version}} -> {{release.next_version}}
+{% if release.changelog %}
+Changes:
+{{release.changelog}}
+{% endif %}
+{% endfor -%}
+"""
+```
+
+Where:
+
+:::warning
+`{{ release.title }}` and `{{ release.changelog }}` may be unset, please use `{% if <variable> %}`
+structures to check for their existence.
+:::
+
+- `{{ releases }}` - is an array with the update information of each package.
+- `{{ release.title }}` - is the changelog title containing a link to the release tag diff.
+- `{{ release.package }}` - is the name of the package being updated.
+- `{{ release.changelog }}` - is the generated changelog.
+- `{{ release.previous_version }}` - is the previous version of the package.
+- `{{ release.next_version }}` - is the version of the package being released.
+
```

Comment:
Can you document why certain fields have the `if` in the example?

Just adding an `(optional)` at the end of the field line should be ok.
Even better: you could explain why these fields are optional.